### ISSUE TITLE
Show the transferred NVDA usage statistics through the settings window in a separate window

### ISSUE THREAD

**Issue by DrSooom** [nvaccess/nvda#8510]
### Path:
NVDA menu » Preferences » Settings... » General
### Actual behaviour:
At the moment there only exists the checkbox called "Allow the NVDA project to gather NVDA usage statistics".
### Expected behaviour:
As this isn't transparent enough for some users, I would like to have an additional button below this checkbox called "Show transferred data...". Clicking on this button will open a new window with an multiline text field and a "Close"-button. Focus is on the first line of this text field which show all information which are transferring to the NVDA Project during the update check.

Well, and please also add a link in the settings window to the website where the user can see those collected statistics. As NVDA is open source, the statistics which are provided by the end user should be fully public for all as well. As they don't include personal data (the IP address isn't stored), this step shouldn't be a problem at all.

#### NVDA version:
NVDA alpha-15554,66186bfb

**Comment by leonardder** [comment]
Note that every user is always prompted once to give consent to this setting. In that prompt, it is perfectly clear what is send to NV Access.

**Comment by josephsl** [comment]
Hi, I think what the original poster is asking here is a list of current values to be sent when NVDA sends additional data as part of update checks. Is this correct?

**Comment by Brian1Gaff** [comment]
Yes I saw that as well. I tend to agree though that just like about, it 
should be recallable somehow just for a new user who may have had it set up 
for them.
 As for access to the data. This assumes somebody has written the scripts to 
make this info format and displayable and of course assuming that the 
country of use is not considered private, ie if you suddenly see a lot of 
Iranian users, might some law enforcement entities become interested?

Personally I don't care where its being used. Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by DrSooom** [comment]
@leonardder: this prompt on the very first startup of NVDA says absolutely nothing about the IP address. No, this prompt actually isn't "perfectly clear" at all.

So I edited the title of this issue and add an update in its description. The user should see not only the types of the metadata, he should also see the value itself to make a suitable decission.

**Comment by josephsl** [comment]
Hi,

2024 update: I think various data points such as synthesizer info and braille display info are noted in the user guide.

Thanks.
